src/components/v5/common/ActionSidebar/partials/ActionButtons.tsx

```diff
@@ -5,7 +5,11 @@ import { useMobile } from '~hooks';
 import Button, { TxButton } from '~v5/shared/Button';
 import { useActionSidebarContext } from '~context/ActionSidebarContext';
 import { ActionButtonsProps } from '../types';
-import { useSubmitButtonDisabled, useSubmitButtonText } from './hooks';
+import {
+  useDisableFields,
```

Comment:
hope it's not too nitpicky, but `useDisableFields` makes it sound like it's going to disable fields, maybe rename to `useIsFieldDisabled`